modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItSqlApiBaseTest.java

```diff
@@ -894,6 +897,102 @@ public void testTimeZoneId(String timeZoneId) {
         assertEquals(nowMillis, tsMillis, deltaMillis);
     }
 
+    @Test
+    public void testEarlyQueryTimeout() {
```

Comment:
all new test work in synchronious manner, but  API should depend on subclasses. Please see another tests how it works

Reply:
Fixed.